src/main/kotlin/gg/octave/bot/commands/music/dj/Shuffle.kt

```diff
@@ -25,4 +28,14 @@ class Shuffle : MusicCog {
         manager.scheduler.shuffle()
         ctx.send("Player has been shuffled")
     }
+
+    @SubCommand(description = "Toggle whether the queue should be shuffled before every play.")
+    fun auto(ctx: Context, option: AutoShuffle) {
+        ctx.manager.scheduler.autoShuffle = option
+
+        val send = if (option == AutoShuffle.OFF) "Playlist will no longer be shuffled automatically."
+        else "Playlist will be shuffled for every song."
```

Comment:
Adjust description to new logic.
```suggestion
        else "Queue will be shuffled when songs are added."
```